pipeline/behavior.py

```diff
@@ -0,0 +1,75 @@
+'''
+Schema of session information.
+'''
+import datajoint as dj
+from pipeline import acquisition
+
+schema = dj.schema('ttngu207_behavior',locals())
+
+        
+@schema
+class TrialType(dj.Lookup):
+    definition = """
+    type_name: varchar(64)
+    ---
+    
+    """
+    contents = [
+            ['Hit'],
+            ['Miss'],
+            ['CR'],
+            ['FA'],
+            ['Stimtrials']
+            ]
+
+@schema
+class TrialSet(dj.Manual):
+    definition = """
+    -> acquisition.Session
+    ---
+    number_of_trials: int
+    time_unit: enum('millisecond','second','minute','hour','day')
+    """
+    class Trial(dj.Part):
+        definition = """
+        -> master
+        trial_idx: int
+        ---
+        -> TrialType
+        pole_trial_condition: enum('Go','NoGo')  # string indicating whether the pole was presented in a ‘Go’ or ‘Nogo’ location
+        pole_position: float                     # is the location of the pole along the anteroposterior axis of the animal in microsteps (0.0992 microns / microstep)
```

Comment:
starting a comment with `"is ..."` wrongly suggest that the field is a boolean value (True/False answer to the comment's question). Simply state what the value is:

```suggestion
        pole_position: float                     # pole location along the AP axis in microsteps (0.0992 microns / microstep)
```

Also if this is given in microsteps, shouldn't this be an integer value? Furthermore, consider already converting the data into that most natural data. So here, you could go ahead convert the values into `microns` from microsteps during the ingestion.

Reply:
This comment in general applies to other attributes below.